org.eclipse.lemminx/pom.xml

```diff
@@ -287,5 +287,11 @@
 			<version>9.4.53.v20231009</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
We don't use mockito for test.

@fbricon @datho7561 do you think it can be a good idea to use mockito?